contracts/Identity.sol

```diff
@@ -64,24 +64,44 @@ contract Identity is KeyManager {
     address to,
     uint256 value,
     bytes memory data
-  )
+  ) onlyAction
   public
   returns (bool success)
   {
-
-    bytes32 key_ = addressToKey(msg.sender);
-    require(
-      keyHasPurpose(key_, ACTION) && _keys[key_].revokedAt == 0,
-      "Requester must have an ACTION purpose"
-    );
-
     // solium-disable-next-line security/no-inline-assembly
     assembly {
-      // check if contract to be call exists
+      // check if contract to be called exists
       if iszero(extcodesize(to)) {
         revert(0, 0)
       }
       success := call(gas, to, value, add(data, 0x20), mload(data), 0, 0)
     }
   }
+  /**
+   * @dev Transfer eth
+   * @param to address of account where to transfer eth.
+   * it can not be a contract. Use execute method in that case
+   * @param value uint256 wei supply for proxy execution
+   * @param data bytes ABI encoded call data
+   */
+  function transferEth(
+    address to,
+    uint256 value,
+    bytes memory data
+  ) onlyAction
+  public
+  returns (bool success)
+  {
+    // solium-disable-next-line security/no-inline-assembly
+    assembly {
+    // Make sure the address is not a contract
+      if gt(extcodesize(to),0) {
+        revert(0, 0)
```

Comment:
Why not calling the `execute` method in that case?

Reply:
I do not understand you quesiton

Reply:
In case a user wants to send ETH to a contract. The user should use the `execute` method.

```suggestion
        success := execute(...)
```

Reply:
Ok. but calling the execute method would result in the same line of code in the end:
` success := call(gas, to, value, add(data, 0x20), mload(data), 0, 0)`

Maybe I am questioning more the if condition itself :)

Reply:
Because I want this method to revert and force the user to use the execute api. Execute is for contract call and transferEth is for account calling. The old execute method worked like this and the idea is to be more clear and have 2 methods.

Reply:
The execute checkIfContract exists was a recommendation from ToB, if I am correct?
I don't want to change that.

In Ethereum accounts and contracts are on the same level. Let say I need to send ETH to a 1000 addresses, why should I care if the other users using an account or a contract.

Maybe I don't know it, and I would need to implement an additional check and switching between `execute `and `transferETH` for my 1000 transactions

Reply:
Yes that what you should do. You should know what addresses you are using. It the same thing if you send some Eth from metamask. The client should know if the address is correct and what is behind it. We could remove the check in the tranferEth and make it like the old execute method but then you could bypass execute and you are losing eth in the case you wanna call a contract and send it eth

Reply:
I understood the Ethereum philosophy in that way:  It shouldn't matter if an address is an account or a contract. This design decision enabled a lot of cool things like multi-sig contracts etc.

In case somebody would use our identity transferETH in another contract without an additional isContract check. It would suddenly matter if an address is an account or a contract.

Reply:
Anyway, it is a more general discussion.  Overall PR looks good. If a `payable` keyword is not needed and ETH is transferred.